src/features/network-modifications/twoWindingsTransformer/creation/twoWindingsTransformerCreation.utils.ts

```
@@ -0,0 +1,344 @@
+    }
+    return tap.terminalRefConnectableId === twt.equipmentId ? REGULATION_TYPES.LOCAL.id : REGULATION_TYPES.DISTANT.id;
+};
+
+export const computeRatioTapChangerRegulationMode = (
+    ratioTapChangerFormValues?: RatioTapChangerCreationDto | TapChangerMapInfos | null
+) => {
+    if (ratioTapChangerFormValues?.isRegulating) {
+        return RATIO_REGULATION_MODES.VOLTAGE_REGULATION.id;
+    } else {
+        return RATIO_REGULATION_MODES.FIXED_RATIO.id;
+    }
```

ESLint

[error] Unnecessary 'else' after 'return'.

(no-else-return)